src/components/Independent/Independent.js

```diff
@@ -14,40 +15,33 @@ import { getOembed } from '../../api-calls';
 import PropTypes from 'prop-types';
 
 
-const Independent = ({ tiktoksrc, addPin, removePin, removeTikTok }) => {
-  const [title, setTitle] = useState('');
-  const [author, setAuthor] = useState('');
-  const [authorLink, setAuthorLink] = useState('');
-  const [thumbnail, setThumbnail] = useState('');
+const Independent = ({
+  cite,
+  title,
+  author_url,
+  author_name,
+  html,
+  data_video_id,
+  thumbnail_url,
+  status_msg,
+  removeTikTok,
+  addPin,
+  removePin
+  }) => {
   const [isPinned, setPin] = useState(false);
-  const [error, setError] = useState('');
 
   const togglePin = () => {
     if (isPinned) {
       setPin(false);
-      return removePin(tiktoksrc);
+      return removePin(data_video_id);
     }
 
     if (!isPinned) {
       setPin(true);
-      return addPin(tiktoksrc);
+      return addPin(data_video_id);
```

Comment:
Can you consolidate these two if statements?  To go along with consolidating the add/remove Pin methods.

Reply:
Good point, referencing my current ternary
<code>
const togglePin = () => { isPinned ? removePin(data_video_id) : addPin(data_video_id) }
</code>
Reassign to the opposite with the bang operator
<code>
const togglePin = () => { isPinned = !isPinned }
</code>